api/python/cellxgene_census/src/cellxgene_census/experimental/util/_csr_iter.py

```diff
@@ -19,12 +19,15 @@ def X_sparse_iter(
     stride: int = 2**16,
     fmt: Literal["csr", "csc"] = "csr",
     use_eager_fetch: bool = True,
+    reindex_sparse_axis: bool = True,
```

Comment:
CC, *FYI only*: @beroy - this use case will need to eventually be ported to fast reindexing and COO conversion, when they are available.